README.md

```diff
@@ -42,29 +42,140 @@ Tools for extracting and editing Coktel Vision game resources.
+* GAME_PATH            : path of the game folder (which contains all .ITK, .STK, ... files)
+* EXTRACTED_GAME_PATH  : the result path / folder after running the extract script
+* DECOMPILED_GAME_PATH : the result path / folder after running the decompiler script
+* TEXTS_GAME_PATH      : the result path / folder after running the texts extractor script
+* GRAPHICS_GAME_PATH   : the result path / folder after running the graphics extractor
+
+### The HELP command
+
+**Each script** comes with an HELP.  
+You can trigger the feature like this:
+```bash
+python -m boozook.SCRIPT_FILE --help
+```
+
+This can help you to understand what type of input, or argument, you need to provide.
+
+### Extract game resources
+To extract the game resources you can use the `archive` script:
+```bash
+python -m boozook.archive GAME_PATH
+```
+
+This will extract, in a folder called `extracted` at the root of the `boozook` folder (the `EXTRACTED_GAME_PATH` folder), 
+all .STK content that has been detected from the `archive` tool.
+
+### Rebuild game resources
+To rebuild the extracted game resources you can use the `archive` script as well, **but** with the `-r` (or `--rebuild`) argument:
+```bash
+python -m boozook.archive -r EXTRACTED_GAME_PATH
+```
+
+> [TODO _ TO_DISCUSS]  
+> This command does not trigger any error **but** does not output any new file.  
+> What is the output of this command?
+
+### Script decompiler
+To decompile a TOT script you can use the `decomp_tot` script:
+```bash
+python -m boozook.codex.decomp_tot EXTRACTED_GAME_PATH/{FILE}.TOT {VERSION}
+```
+
+> [TODO _ TO_DISCUSS]
+> Seems like there is currently an issue with the script decompiler:
+> 
+> python -m boozook.codex.decomp_tot  
+> > C:\Users\Test\boozook\extracted\Test.TOT 52    
+> > usage: decomp_tot.py [-h] [--lang LANG] [--keys] [--exported] directory [totfiles ...] {48,49,50,51,52}  
+> > decomp_tot.py: error: argument version: invalid choice: '52' (choose from 48, 49, 50, 51, 52)
+>
+> This is the same error for any argument provided, need to check what is the real error provided by the Python script
+
+where:
+* `{FILE}` is the TOT file (name) you want to decompile.
+* `{VERSION}` is the script version to decompile.
+
+This will extract, in a folder called `script` (the `DECOMPILED_GAME_PATH`) at the root of the `boozook` folder,
+all script content of the specified .TOT file.
+
+### Text extraction
+To extract text of a script file you can use the `text` script:
+```bash
+python -m boozook.text EXTRACTED_GAME_PATH/{FILE}.STK
+```
+
+This will extract, in a folder called `texts` at the root of the `boozook` folder (the `TEXTS_GAME_PATH` folder).
+
+### Text rebuild
+To recompress the modified texts, back to an archive, you can use the `test` script as well  **but** with the `-r` (or `--rebuild`) argument:
+```bash
+python -m boozook.text -r GAME_PATH
+```
+
+> [TODO _ TO_DISCUSS]  
+> How to use the rebuild tool ?  
+> Do we have to point to the texts output directory?
```

Comment:
we should put the extracted text to the texts folder as before.

Reply:
Done!
And I removed the TODO statement